### PR TITLE
Renamed 'core/Location' to 'core/history'

### PR DESCRIPTION
**Pull request by rusjoan**
npm history module v2.0.2

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ function run() {
   FastClick.attach(document.body);
 
   // Re-render the app when window.location changes
-  const removeLocationListener = Location.listen(location => {
+  const removeLocationListener = history.listen(location => {
```
Comment (koistya) [inline src/client.js]: removeHistoryListener

**Comment by koistya** [comment]
Thanks! For review @frenzzy

**Comment by rusjoan** [comment]
@frenzzy check please

**Comment by frenzzy** [comment]
Looks good to me! Closes #570

**Pull request merged**